cli/commands/usage.go

```diff
@@ -87,6 +87,8 @@ defaults to ./instances.yml ("cfg" in .cartridge.yml)`
 	logFollowUsage = `Output appended data as the log grows`
 
 	stopForceUsage = `Force instance(s) stop (sends SIGKILL)`
+
+	disableLogPrefixUsage = `Disable instance(s) prefix in logs`
```

Comment:
`instance(s) name prefix`?

Or maybe simple `Disable prefix in logs when run interactively`?